Xamarin.Forms.FlexLayout/Xamarin.FlexLayout.Shared/Xamarin.Forms.FlexLayout.cs

```diff
@@ -0,0 +1,313 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace Xamarin.Forms
+{
+	public partial class FlexLayout : Layout<View>
+	{
+		static readonly BindableProperty FlexNodeProperty = BindableProperty.CreateAttached(_FlexNodePropertyName, typeof(IFlexNode), typeof(FlexLayout), null);
+
+		public static readonly BindableProperty IsIncludedProperty = BindableProperty.CreateAttached(_FlexIsIncludedPropertyName, typeof(bool), typeof(FlexLayout), true);
+
+		public static readonly BindableProperty OrderProperty = BindableProperty.CreateAttached(_FlexOrderPropertyName, typeof(int), typeof(FlexLayout), 0);
+
+		public static readonly BindableProperty GrowProperty = BindableProperty.CreateAttached(_FlexGrowPropertyName, typeof(double), typeof(FlexLayout), 0.0, propertyChanging: (s, o, n) => { if (GetNode((View)s) == null) return; GetNode((View)s).FlexGrow = (float)n; });
```

Comment:
why are all BP using `propertyChanging` instead of `porpertyChanged` ? `propertyChanging` is usually to perform cleanup, and actual changes happens in `PropertyChanged`